projects/igniteui-angular/src/lib/grids/filtering/grid-filtering.service.ts

```diff
@@ -323,6 +326,29 @@ export class IgxFilteringService implements OnDestroy {
         }
     }
 
+    public resolveFilteredSortedData(): any[] {
```

Comment:
This is gonna be removed anyway since it is only moved to the service by us due to the class hierarchy changes from the grid component. It is already removed in the master branch where it previous place was.

Reply:
@skrustev Then remove it.